src/main/perl/Lock.pm

```diff
@@ -9,21 +9,22 @@ use strict;
 use CAF::Object;
 use CAF::Reporter;
 
-use LC::Exception qw (SUCCESS throw_error);
+use LC::Exception qw(SUCCESS throw_error);
 use FileHandle;
-#use Proc::ProcessTable;
+use Fcntl qw(:flock);
 
 use Exporter;
 use vars qw(@ISA @EXPORT @EXPORT_OK);
 
 @ISA = qw(CAF::Reporter CAF::Object Exporter);
```

Comment:
There's something seriously wrong with this. `CAF::Lock` is it's own reporter? i opened #146 to follow up as soon as this is merged.